File: ITMrutaN/warships.py
while True:
    try:
        tipoNaveTobby, cantidadTobby = map(str,input().split())
        tipoNaveNab, cantidadNab = map(str,input().split())

        cantidadTobby = int(cantidadTobby)
        cantidadNab = int(cantidadNab)
        if(tipoNaveTobby==tipoNaveNab):
            if(cantidadTobby>cantidadNab):
                print("Tobby")
            elif(cantidadTobby<cantidadNab): 
                print("Naebbirac")
            elif(cantidadNab==cantidadTobby):
                print("Draw")

        if(tipoNaveTobby=="Battleships"):
            if(tipoNaveNab == "Destroyers" and (cantidadNab >= 2*cantidadTobby)):
                print("Naebbirac")
            elif tipoNaveNab == "Destroyers":
                print("Tobby")


        if(tipoNaveNab == "Battleships"):
            if(tipoNaveTobby == "Destroyers" and (cantidadTobby >= 2*cantidadNab)):
                print("Tobby")
            elif tipoNaveTobby == "Destroyers":
                print("Naebbirac")


        if(tipoNaveTobby=="Destroyers"):
            if(tipoNaveNab == "Cruisers" and (cantidadNab >= 2*cantidadTobby)):
                print("Naebbirac")
            elif(tipoNaveNab == "Cruisers"):
                print("Tobby")

        if(tipoNaveNab=="Destroyers"):
            if(tipoNaveTobby == "Cruisers" and (cantidadTobby >= 2*cantidadNab)):
                print("Tobby")
            elif(tipoNaveTobby == "Cruisers"):
                print("Naebbirac")


        if(tipoNaveTobby=="Cruisers"):
            if(tipoNaveNab == "Battleships" and (cantidadNab >= 2*cantidadTobby)):
                print("Naebbirac")
            elif(tipoNaveNab == "Battleships"):
                print("Tobby")

        if(tipoNaveNab=="Cruisers"):
            if(tipoNaveTobby == "Battleships" and (cantidadTobby >= 2*cantidadNab)):
                print("Tobby")
            elif(tipoNaveTobby == "Battleships"):
                print("Naebbirac")
    except:
        break

File: ITMrutaN/warships.cpp
#include<iostream>
#include<string>

using namespace std;

int main(){
    ios_base::sync_with_stdio(false);
    cin.tie(NULL); cout.tie(NULL);
    int T,N;
    string Naves,Naves1;
    cin>>Naves>>T; //Tobby
    cin>>Naves1>>N; //Nae
    if (Naves1==Naves){
        if(T>N){
            cout<<"Tobby\n";
        }else if(T<N){
            cout<<"Naebbirac\n";
        }else if(N==T){
            cout<<"Draw\n";
        }
    }
    else if(Naves=="Battleships"){
        if(Naves1=="Destroyers" && T/2>=N){
            cout<<"Naebbirac\n";
        }else{
            cout<<"Tobby\n";
        }
    }
    else if(Naves=="Destroyers"){
        if(Naves1=="Cruisers" && T/2>=N){
            cout<<"Naebbirac\n";
        }else{
            cout<<"Tobby\n";
        }
    }
    
    else if(Naves=="Cruisers"){
        if(Naves1=="Battleships" && T/2>=N){
            cout<<"Naebbirac\n";
        }else{
            cout<<"Tobby\n";
        }
    }
}